### PR TITLE
New portal monster

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
Content "New portal monster"

#### Purpose of change

Add a new portal monster

#### Describe the solution

The solution is kinda wonky, but: the monster, that can see the player no matter what (and player also can see it no matter what) appears somewhere close to the reality bubble border, and slowly approach the player. When it touch the player, it gives him an effect, and special EoC make a short term hallucination attacks for a 10 days

#### Issues

right now im waiting for #59398 to be merged, to use it in this monster (thank you Venera3, its the main reason i am able to finish it)
and i still think what is better: use the current implementation, with monster, that emit clairvoyance field around (so i can also show not only the monster itself, but also monsters nearby), or just use the `ALWAYS_VISIBLE`  flag, which do the same, but does not allow to show the territory nearby (and also works only on curses due the #59504)

#### Additional context

i guess i need to make some more clairvoyance portal storm effects, because they looks cool imo
The magiclysm label was added because i moved the magiclysm clairvoyance field to vanilla

GIF 24 07 2022 15-12-38

### CODE REVIEW

**Comment by anoobindisguise** [comment]
Does this enemy respect use of portal_proof effects like 5-pt anchor and phase immersion? It does seem like it should if it doesn't.

**Comment by GuardianDll** [comment]
its not, and i probably add it

File: data/json/monster_special_attacks/spells.json [hunk 1]
```diff
@@ -578,9 +578,9 @@
     "valid_targets": [ "hostile", "ally", "self" ],
     "effect": "attack",
     "shape": "blast",
-    "damage_type": "biological",
-    "min_damage": 9999,
-    "max_damage": 9999
+    "damage_type": "pure",
```
Comment (Maleclypse) [inline data/json/monster_special_attacks/spells.json:581]: Was there an issue where boomers weren't dying?

**Review by Maleclypse — COMMENTED** [review]
I see the why the changes to the boomer spell now.   Cool.  I would make the max duration 3 days and each attack increment that duration by 2 to four hours or even 8 hours if this can only appear once per portal storm.   But 10 days per hit would be where you could literally have this until the next portal storm and get stuck hallucinating forever.  And this one isn't treated by meds is it?

**Comment by GuardianDll** [comment]
Yeah, the time can be decreased easily, but the monster itslef has speed 25, which make it pretty easy to just escape it

**Comment by ANickelN** [comment]
Is this intended to deafen the player as well?

**Comment by GuardianDll** [comment]
No, its because the spell attack has lack of SILENT flag

**Pull request merged**